index.js

```diff
@@ -3,7 +3,8 @@ const exec = require('@actions/exec');
 const tc = require('@actions/tool-cache')
 async function run() {
     try {
-        const porterURL = "https://cdn.deislabs.io/porter/latest/install-linux.sh";
+        let porterVersion = core.getInput("porter_version");
+        const porterURL = `https://cdn.deislabs.io/porter/${porterVersion}/install-linux.sh`;
```

Comment:
Oh! I just realized that we changed the cdn since you submitted this PR. I'm so sorry that this fell through the cracks with holidays and team changes.

```suggestion
        const porterURL = `https://cdn.porter.sh/${porterVersion}/install-linux.sh`;
```